packages/obojobo-express/routes/editor.js

```diff
@@ -1,43 +1,62 @@
 const express = require('express')
 const router = express.Router()
 const db = oboRequire('db')
-const { requireCanViewEditor } = oboRequire('express_validators')
+const { requireCanViewEditor, requireCurrentDocument } = oboRequire('express_validators')
 
-// Display the Document Editor
+const displayEditorPicker = (req, res) => {
```

Comment:
We're now using "picker" for the LTI Picker so this is a little confusing. Would rather this be called displayDraftManager, and refer to the "editor picker" as the "Draft manager".

I think it's okay to leave it for now however. I know the existing code referred to it as the editor picker, so it's not really an issue with this PR.